theano/configdefaults.py

```diff
@@ -146,6 +146,13 @@ def filter(val):
     in_c_key=False)
 
 
+AddConfigVar(
+    'enable_initial_driver_test',
+    "Tests the nvidia driver when a GPU device is initialized.",
+    BoolParam(True, allow_override=False),
+    in_c_key=False)
+
+
```

Comment:
Thinking about it, I would probably have put the option in the `cuda` section of the configuration, rather than at the root.
If you're willing to make that change before the flag gets used by other people, that would be cool, otherwise it's not that important.